projects/plugins/jetpack/_inc/lib/markdown/extra.php

```diff
@@ -2047,7 +2047,8 @@ function _hashHTMLBlocks_inMarkdown($text, $indent = 0,
 		return array($parsed, $text);
 	}
 	function _hashHTMLBlocks_inHTML($text, $hash_method, $md_attr) {
-	#
+	$base_tag_name_re = null;
+  #
```

Comment:
WTF on the indentation here?

As seems usual with this fix, this seems to either be masking an actual logic error or to be a failure in code path analysis.